### PR TITLE
feat: adopt 6RS prettier standard (CUR-1069)

### PR DESCRIPTION
**Pull request by mgabeler-lee-6rs**
with strictness updates from https://github.com/6RiverSystems/eslint-config-6river/pull/148

### CODE REVIEW

File: packages/common/package.json [hunk 1]
```diff
@@ -71,6 +71,10 @@
     "nyc": "13.0.1",
     "sinon": "7.5.0",
     "source-map-support": "0.5.16",
-    "typescript": "3.9.7"
+    "typescript": "3.9.7",
+    "@6river/prettier-config": "1.0.37",
+    "eslint-config-prettier": "8.2.0",
+    "eslint-plugin-import": "2.22.1",
+    "prettier": "2.2.1"
```
Comment (AdamVig) [inline packages/common/package.json:78]: These should be alphabetized with the rest of the dependencies in this list, shouldn't they?

**Review by AdamVig — COMMENTED** [review]
I think we should add a commit hook for Prettier ([docs](https://rushjs.io/pages/maintainer/enabling_prettier/#enabling-the-git-hook)).

File: packages/common/package.json (hunk 1 above)
Comment (mgabeler-lee-6rs) [reply packages/common/package.json:78]: ugh, `rush` used to sort package.json entries automatically, dunno wtf happened to that behavior

Comment (mgabeler-lee-6rs) [reply packages/common/package.json:78]: apparently this was an intentional change in rush: https://github.com/microsoft/rushstack/pull/1710

with now people asking them to change it back :) https://github.com/microsoft/rushstack/issues/2496

Comment (mgabeler-lee-6rs) [reply packages/common/package.json:78]: ran `npx sort-package-json` for now

**Comment by mgabeler-lee-6rs** [comment]
> I think we should add a commit hook for Prettier ([docs](https://rushjs.io/pages/maintainer/enabling_prettier/#enabling-the-git-hook)).

I'm not sure how I feel about that. I find commit hooks can get very frustrating very quickly, esp. when it's not obvious how to skip them. Even the commitlint hook gets irritating because you then can't commit "normally" from within a pristine repo, you _must_ `rush install` for that to work "normally", and if you're fighting problems with `rush`, that rapidly gets rage inducing.

I'm willing to take that hit for commit messages, because they are much more costly to fix after the fact. But for the committed code ... CI will fail if your formatting isn't proper, and you can run the linter yourself beforehand. I feel like that starts a slope towards "you can't commit unless CI passes locally", which I personally find maddening.

So ... in total ... I'm willing to discuss, but I have feels against it, and I don't think this PR should be tied to the outcome of that discussion :)

**Review by AdamVig — APPROVED** [review]
> Even the commitlint hook gets irritating because you then can't commit "normally" from within a pristine repo, you must rush install for that to work "normally"

I think Rush's "autoinstaller" feature should help with that, but obviously only to an extent.

> I feel like that starts a slope towards "you can't commit unless CI passes locally", which I personally find maddening.

I heartily agree with this sentiment, yeah. I like your thinking about only using commit hooks for things that are costly to fix after the fact.

I look at code formatting on a different plane: it's easy to fix, but it creates useless commits ("fix: run Prettier") if you don't catch it until CI runs. Obviously, as evils go, this is a minor one. :slightly_smiling_face:

The editor integration is really the key piece of implementing Prettier successfully. The commit hook is mostly useful for people have their editors misconfigured, so I don't see this as a big deal.

**Pull request merged**